.eslintrc.js

```diff
@@ -122,4 +115,9 @@ module.exports = {
       },
     },
   },
+  overrides: [
```

Comment:
it works when run `yarn lint`, without this configuration lint does not run on any file

Reply:
It sounds like it would be better to modify the lint command itself (`--fix src/**/*.{...}`), because this is equivalent to cross-file and implicitly associated coupling. This would make it difficult for people who haven't seen this comment to think of this, leading to maintenance issues with related problems.